tests/gameBoardModule.test.js

```diff
@@ -0,0 +1,24 @@
+import { gameBoardModule } from '../main';
+
+describe('checking gameboard', () => {
+  const board = ['', '', '', '', '', '', '', '', ''];
+  const newBoard = ['', 'X', '', '', '', '', '', '', ''];
+
+  test('should check for winning combos', () => {
+    expect(gameBoardModule.checkWinCombo()).toBe(null);
+  });
```

Comment:
- [ ] We need to implement more tests for the checkWinCombo functionality. You only check for the negative case. We need to test if it's returning true when the board has any of the possible winning cases. ( horizontal, vertical, and diagonal). Meaning, test for horizontal winning cases, vertical winning cases and of course, diagonal winning cases,

- [ ] We also need to implement a test case for the Tie situation.